crates/phactory/src/system/mod.rs

```diff
@@ -358,6 +358,22 @@ impl WorkerStateMachineCallback for WorkerSMDelegate<'_> {
     }
 }
 
+#[derive(
+    Serialize, Deserialize, Clone, derive_more::Deref, derive_more::DerefMut, derive_more::From,
+)]
+#[serde(transparent)]
+pub(crate) struct WorkerIdentityKey(#[serde(with = "more::key_bytes")] sr25519::Pair);
+
+#[cfg(feature = "shadow-gk")]
```

Comment:
Interesting. Should we add more comments here? Something like: `By mocking the public key of the identity key pair, we can pretend to be the first Gatekeeper on Khala for "shadow-gk" simulation.`

Reply:
Done